tree/20_cfe_basics/log_rudder.cf

```diff
@@ -437,13 +433,8 @@ body classes rudder_common_classes(prefix)
 #
 body classes rudder_common_classes_persist(prefix, persist)
 {
-    promise_kept     => { "${prefix}_kept"    , "${prefix}_ok" };
-    promise_repaired => { "${prefix}_repaired", "${prefix}_ok" };
-    repair_failed    => { "${prefix}_failed"  , "${prefix}_error" };
-    repair_denied    => { "${prefix}_denied"  , "${prefix}_error" };
-    repair_timeout   => { "${prefix}_timeout" , "${prefix}_error" };
-
-    persist_time     => ${persist};
+    inherit_from => classes_generic("${prefix}");
```

Comment:
why not inherit from rudder_common_classes ?

Reply:
changed, it makes more sense indeed.